src/dwarf.cc

```diff
@@ -2136,9 +2163,15 @@ static void ReadDWARFDebugInfo(
       });
 
       // low_pc == 0 is a signal that this routine was stripped out of the
-      // final binary.  Skip this DIE and all of its children.
```

Comment:
You might want to also check int max (in the appropriate size (32 bit or 64 bit depending on the address size in use in this DWARF)) - since that's hopefully the linker tombstone value we'll use in the future. (It's been implemented in clang/llvm, but not rolled out far yet).

But not high priority - more future proofing.

Reply:
Done (this was previously implemented in https://github.com/google/bloaty/pull/211, but I agree this is a better place to check it).